README.md

```diff
@@ -529,6 +529,16 @@ RUST_LOG=info cargo run -p zome_call_single_value -- --duration 60 --connection-
 
 #### Running Wind Tunnel Scenarios with Nomad
 
+##### Job Variants
+
+A Nomad job will be deployed in different "job variants" depending on its context. A job variant defines the nomad variables applied to the job.
```

Comment:
I'm expecting we will want to add additional variants for infrequently run jobs or durations. For example running a scenario with an additional 1000 nodes spun up from an api request to digital ocean.